client/hotrod-client/pom.xml

```diff
@@ -603,7 +603,7 @@
             <artifactId>karaf-maven-plugin</artifactId>
             <configuration>
                <features>
-                  <feature>infinispan-client-hotrod,infinispan-client-hotrod-with-query,hotrod-client,hotrod-client-with-query</feature>
+                  <feature>infinispan-client-hotrod,infinispan-client-hotrod-with-query</feature>
```

Comment:
```suggestion
                  <feature>infinispan-client-hotrod</feature>
                  <feature>infinispan-client-hotrod-with-query</feature>
```

Reply:
fair enough

Reply:
I made that change. But did not get to investigate the CI failures as I'm trapped into something else. I saw them on master also, as you said.